### DISCUSSION TITLE
How do I align headers inside of columns Revealjs

### DISCUSSION THREAD

**Discussion by victorfeagins** [quarto-dev/quarto-cli#7787] [Q&A] [answered]
### Description

When I render this the Cats column is slightly ajar to the right how do I adjust so it is left aligned similar to the dogs column

````qmd
---
title: "test"
format: revealjs
---

## Title here
:::: {.columns}

::: {.column width="50%" .fragment}

### Dogs
- likes bones
- 4 legs
:::

::: {.column width="50%" .fragment}
### Cats
- likes fish
- 4 legs
:::

::::

````

**Comment by mcanouil** [comment]
Use CSS.
To know what to target, use your favourite browser developer mode "inspect" and use your mouse pointer to target elements and see the associated elements, then tweak it live to finally use that in your document.

There are several examples of CSS customisation in other threads.

**Comment by victorfeagins** [comment]
I inspected the headers and they seems to be in the div columns though this rule did not fix the problem.:
````{css}
.columns {
   margin-left: 0px;
}
````
Instead I adjusted the h3 tag directly and that seemed to work.
````{qmd}
---
title: "test"
format: revealjs
---

## Title here
:::: {.columns}

::: {.column width="50%" .fragment}

### Dogs
- likes bones
- 4 legs
:::

::: {.column width="50%" .fragment}
<h3 id="cats" style="margin-left:0px;">Cats</h3>
- likes fish
- 4 legs
:::

::::
````

Does anyone know what div to change?

**Reply by mcanouil** [reply, marked as answer]
Why did you not target the element highlighted by the inspect mode?

https://github.com/quarto-dev/quarto-cli/assets/8896044/9b424c58-90f2-42cc-a65e-9427aa7ac9af

````qmd
---
title: "test"
format: revealjs
include-in-header:
  text: |
    <style type="text/css">
    .reveal .columns>.column:last-child>:not(ul,ol) {
      margin-right: 0;
      margin-left: 0;
    }
    </style>
---

## Title here

:::: {.columns}

::: {.column width="50%" .fragment}
### Dogs
- likes bones
- 4 legs
:::

::: {.column width="50%" .fragment}
### Cats
- likes fish
- 4 legs
:::

::::
````

**Reply by victorfeagins** [reply]
I am not familiar with CSS or the inspector tool. Thank you for showing the process of investigating this.

**Reply by mcanouil** [reply]
Unfortunately, to get customisation of HTML based, there is no way around CSS.
There are pretty good resources online to learn the basics.
